conanfile_py.rst

```diff
@@ -88,13 +90,13 @@ If you are building your project with CMake, edit your ``conanfile.py`` and add
    
       def build(self):
          cmake = CMake(self.settings)
-         self.run('cmake "%s" %s' % (self.conanfile_directory, cmake.command_line))
-         self.run('cmake --build . %s' % cmake.build_config)
+         cmake.configure(self)
```

Comment:
Sure! Agree the "command line version" should be mentioned, it's still the one currently used by most packages.
I can do an update maybe this evening or tomorrow.